### DISCUSSION TITLE
Expand Client and Domain Lists

### DISCUSSION THREAD

**Discussion by mikebol** [AdguardTeam/AdGuardHome#2816] [Ideas]
New to using AdGuard but up and running smoothly so far.  One thing I've finding straight away, the client and domains list on the dashboard are simply too narrow.  I believe there's more information that would be useful to add to these lists and the fully qualified names are simply exceeding the available with.

**I'd like to see these lists available full width on a page rather than a quick snapshot.**  In my opinion, Dashboards should be summaries and graphs and tables.  These lists cause me to look for additional detail that's simply not readily available without querying the log for everything.  Comparing this use case with Pi-hole, I find their dashboards easier to use and find information on "what's going on".

Thanks for considering...

**Comment by ainar-g** [comment]
@ameshkov, should we add this to the new design tracking issue?

**Reply by ameshkov** [reply]
Well, they're not that narrow in the new design.
